dev/ci/scripts/ci-common.sh

```diff
@@ -46,12 +49,18 @@ export PATH="$COQBIN:$PATH"
 # Rocq's tools need an ending slash :S, we should fix them.
 export COQBIN="$COQBIN/"
 
-ls -l "$COQBIN"
+if [ -z "${CI_QUIET}" ];
+then
+  ls -l "$COQBIN"
```

Comment:
we could delete these random `ls`, I don't think they're particularly useful

Reply:
Indeed but I don't think this is directly related to this PR though.